### ISSUE TITLE
Slack/Members cleanup

### ISSUE THREAD

**Issue by 59** [nodejs/inclusivity#81]
tl;dr: there's a lot of members from months ago that haven't been active at all. now that the WG is ratified, we should make an effort to remove those from the team.

__purpose__

this WG started out months ago with a few initial members. we can define those as the 'first generation'. started by @ceejbot and @nebrius, they had one meeting, but unfortunately, activity decreased, leading  up to simply not existing.

then, one fateful day, @Trott opened #7, which garnered quite a bit of attention and brought some people that were willing to bring this project back up again on board. these are mostly the ones active in the WG today, hereby referred to as the 'second generation'. 

the current situation is this: the first generation is mostly inactive, but still remains on the team and slack (mostly for legacy purposes).

__focus and proposal__

what would be the optimal outcome from this issue would be a way to contact the first generation and ask them if they're still on board. then, we need some buffer time to wait for responses, after which we'll start removing (?) members from the org team and our private slack.

this would be the primary focus of this issue, so please keep it on topic. if there are any possible secondary outcomes to spring from this, feel free to address them though.

**Comment by sup** [comment]
addendum: regarding the history of this WG, @nebrius is writing up a more detailed piece

**Comment by ashleygwilliams** [comment]
a thought: i don't think we need to remove members so much as create a public and private space. the current (2nd gen group) should have access to the private space. everyone can remain in the public space. thoughts?

**Comment by sup** [comment]
@ashleygwilliams sounds reasonable, although we need to define the medium of said public space, slack/gitter/etc

**Comment by ashleygwilliams** [comment]
i'd say slack still. multiple chat programs is a pita, imo. open to others thoughts tho, obvi.

**Comment by juliepagano** [comment]
+1 on having a public and private space. Having a public space will make it easier for us to include collaborators. Slack seems fine to me.

**Comment by nebrius** [comment]
Do we have thoughts on how we want to make the public space public? Will this be a "anyone can join" sort of thing? Will we publish the contents of the public rooms to GitHub, but still require that anyone who joins be a collaborator or member? Something else?

**Comment by nebrius** [comment]
Can we close this now that the slack inviter is live and we've moved to the new slack?

**Comment by 59** [comment]
yep!
